presto-spi/src/main/java/com/facebook/presto/spi/function/SqlFunctionId.java

```diff
@@ -86,4 +84,29 @@ public String toString()
                 .collect(joining(", "));
         return format("%s(%s)", functionName, arguments);
     }
+
+    @JsonValue
+    public String toJsonString()
+    {
+        return format("%s;%s", functionName.toString(), argumentTypes.stream().map(TypeSignature::toString).collect(joining(";")));
```

Comment:
I kinda prefer this to be `function_name(type,type,type...)`. Looks more like a function signature.

Reply:
The toString() does look like that, so that is what you would see when debugging. This is purely to serialize/deserialize, so I wanted to go with the simplest foolproof solution.

Reply:
We're using SqlFunctionId.getValue, which is essentially calling toString() when storing it into MySQL. We should consolidate the two.

Reply:
Do you want me to replace all the instances of getId() with toJsonString()?